### ISSUE TITLE
Fix pointwise_ops.test_torch_real

### ISSUE THREAD

**Issue by ivy-leaves** [unifyai/ivy#17157]
|   |   |
|---|---|
|numpy|<a href="https://github.com/unifyai/ivy/actions/runs/6382227308/job/17320366501"><img src=https://img.shields.io/badge/-success-success></a>
|jax|<a href="https://github.com/unifyai/ivy/actions/runs/6382227308/job/17320366501"><img src=https://img.shields.io/badge/-success-success></a>
|tensorflow|<a href="https://github.com/unifyai/ivy/actions/runs/6382227308/job/17320366501"><img src=https://img.shields.io/badge/-success-success></a>
|torch|<a href="https://github.com/unifyai/ivy/actions/runs/6382227308/job/17320366501"><img src=https://img.shields.io/badge/-success-success></a>
|paddle|<a href="https://github.com/unifyai/ivy/actions/runs/6382227308/job/17320366501"><img src=https://img.shields.io/badge/-failure-red></a>

**Comment by Gajendran786thee777** [comment]
python
import torch
from torch import nn
from torch.onnx import export

# Define your PyTorch model
class MyModel(nn.Module):
    def __init__(self):
        super(MyModel, self).__init__()
        self.fc1 = nn.Linear(3, 5)
        self.fc2 = nn.Linear(5, 2)

    def forward(self, x):
        x = torch.relu(self.fc1(x))
        x = self.fc2(x)
        return x

# Create an instance of your model
model = MyModel()

# Convert your model to ONNX format
dummy_input = torch.randn(1, 3)
onnx_filename = "my_model.onnx"
export(model, dummy_input, onnx_filename)

python
import onnxruntime as ort
from onnxruntime_tools import optimizer

# Load your model in ONNX format
onnx_model = onnx.load("my_model.onnx")

# Optimize your model
optimized_model = optimizer.optimize_model(onnx_model)

# Save your optimized model
optimized_onnx_filename = "my_model_optimized.onnx"
onnx.save(optimized_model, optimized_onnx_filename)

# Run your model using the ONNX Runtime
ort_session = ort.InferenceSession(optimized_onnx_

**Comment by ivy-leaves** [comment]
Hi there,

Just to let you know, this issue has been depreciated. However, new issues related to any failing tests will be created very soon, marked distinctly with the `Failing Test` label.

Kindly stay tuned for the relevant announcements on our [Discord server](https://discord.com/channels/799879767196958751/983684937712558080) in the coming week.

Thank you for your understanding and cooperation.
